canister_ids.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
we don't have this file before 😨?

Reply:
nope lmao, makes things nice to interact with the canisters 👌🏼